test/acceptance/tests.py

```diff
@@ -75,6 +75,9 @@ class OpenAssessmentTest(WebAppTest):
         'full_workflow_staff_required':
             u'courses/{test_course_id}/courseware/'
             u'8d9584d242b44343bc270ea5ef04ab03/0b0dcc728abe45138c650732af178afb/'.format(test_course_id=TEST_COURSE_ID),
+        'multiple_ora':
+            u'courses/{test_course_id}/courseware/'
+            u'8d9584d242b44343bc270ea5ef04ab03/0b0dcc728abe45138c650732af178afb'.format(test_course_id=TEST_COURSE_ID),
```

Comment:
Is there a reason for re-using the location of the `full_workflow_staff_required` problem? I feel that this would be better served as its own problem.

Also note that if you've changed the test course locally/on your sandbox for this test, your PR needs to include the changes to https://github.com/edx/edx-ora2/blob/master/scripts/data/test-course.tar.gz. (If you haven't changed the test course, ignore the rest of this comment.) This could be why you're having trouble with tests locally - if your local course is changed it could affect pre-existing tests.

Once you've got the course exactly as you like, export it and save to `scripts/data/test-course.tar.gz` in your ora2 git branch. When you push that change up, please note exactly what changed in an inline comment on the PR, as the file is compressed and it isn't easy to see what's changed by looking at the diff.

Before merging, please ensure that:
- [ ] The sandbox you use to run acceptance tests in jenkins has your course updates.
- [ ] A jenkins run of acceptance tests using this branch and pointed at that sandbox succeeds.
- [ ] This PR includes any changes you've made to the test course (such as adding a multiple assessment page)